src/main/java/org/apache/commons/text/names/Name.java

```diff
@@ -26,14 +26,16 @@
 public final class Name {
 
     private final String leadingInitial;
+    private final String salutation;
     private final String firstName;
     private final String nickName;
     private final String middleName;
     private final String lastName;
     private final String suffix;
 
-    Name(String leadingInitial, String firstName, String nickName, String middleName, String lastName, String suffix) {
+    public Name(String leadingInitial, String salutation, String firstName, String nickName, String middleName, String lastName, String suffix) {
```

Comment:
Hi @tmackenzie I reverted this change, and also the final in HumanNameParser locally; all test still pass. But I'm not able to push this change. The GitHub repository for commons-text is a read-only mirror of the Git repository at Apache.

So I created a PR against your repository https://github.com/VividSeats/commons-text/pull/4